src/authService.js

```diff
@@ -7,9 +7,35 @@ import {BaseConfig} from './baseConfig';
 
 @inject(Authentication, BaseConfig)
 export class AuthService {
+  /**
+   * The Authetnication instance that handles the token
+   * @type {Authentication}
+   */
+  authentication;
+
+  /**
+   * The Config instance that contains the current configuration setting
+   * @type {Config}
+   */
+  config;
+
   constructor(authentication, config) {
     this.authentication = authentication;
     this.config         = config;
+
+    // get token stored in previous format over
+    const oldStorageKey = config.tokenPrefix
+                        ? config.tokenPrefix + '_' + config.tokenName
+                        : config.tokenName;
+    const oldToken = authentication.storage.get(oldStorageKey);
+
+    if (oldToken) {
```

Comment:
we changed the key and format of the storage. so, in case an enduser, is logged in (old format) but now runs the updated app requiering new format, it gets little messy. that code just looks for the old format and resaves it in the new one so the enduser won't notice something has changes